ethcore/src/client/config.rs

```diff
@@ -99,6 +99,9 @@ pub struct ClientConfig {
 impl Default for ClientConfig {
 	fn default() -> Self {
 		let mb = 1024 * 1024;
+		// Note/TODO: the defaults here are overridden by the command line
+		// argument parser and changes to defaults here must be performed "over
+		// there" as well (see `cli/mod.rs`).
```

Comment:
hmm, I think the long-term solution would be to get rid of `cli/mod.rs` in favor of `structopt`

Reply:
YES. (Hence the 'todo' in my note.)